expiriments_in_progress/past_experiments_and_results

```diff
@@ -0,0 +1,7 @@
+# Past experiments and their results:
```

Comment:
TOL: Perhaps instead of recording this as its own artifact, we could leverage the PR process as a way to capture point-in-time information about what decisions/changes we're making as a result of experiments. That way, we could look through the PR log to find out what happened in the past and why.